hack/verify-profiles.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+
+${SCRIPT_ROOT}/hack/update-profiles.sh --verify
```

Comment:
Is this helper a hack to support autocompletion?

Reply:
just to make it easy for someone to run the "verify" step without having to know the details of how the update-profiles.sh script works